examples/platform/esp32/BLEdeinit.h

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#ifndef BLE_DEINIT
+#define BLE_DEINIT
+
+#if CONFIG_BT_NIMBLE_ENABLED
+#include "esp_log.h"
+#include "esp_nimble_hci.h"
+#include "nimble/nimble_port.h"
+
+static const char * BLE_TAG = "BLE Deinit";
+#endif
+
+void deinitBLE()
+{
+#if CONFIG_BT_NIMBLE_ENABLED
+    int ret = nimble_port_stop();
```

Comment:
Instead of multiple ifdefs, should this entire file be under `CONFIG_BT_NIMBLE_ENABLED`?